2022-02-23 17:34:44,519 INFO    MainThread:3588 [wandb_setup.py:_flush():71] setting env: {}
2022-02-23 17:34:44,520 INFO    MainThread:3588 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-23 17:34:44,520 INFO    MainThread:3588 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220223_173444-3fpnma8r/logs/debug.log
2022-02-23 17:34:44,520 INFO    MainThread:3588 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/testing/wandb/run-20220223_173444-3fpnma8r/logs/debug-internal.log
2022-02-23 17:34:44,520 INFO    MainThread:3588 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f342392bb10>
2022-02-23 17:34:44,520 INFO    MainThread:3588 [wandb_init.py:init():404] calling init triggers
2022-02-23 17:34:44,521 INFO    MainThread:3588 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-23 17:34:44,521 INFO    MainThread:3588 [wandb_init.py:init():460] starting backend
2022-02-23 17:34:44,521 INFO    MainThread:3588 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-23 17:34:44,568 INFO    MainThread:3588 [backend.py:ensure_launched():216] starting backend process...
2022-02-23 17:34:44,610 INFO    MainThread:3588 [backend.py:ensure_launched():222] started backend process with pid: 3628
2022-02-23 17:34:44,612 INFO    MainThread:3588 [wandb_init.py:init():469] backend started and connected
2022-02-23 17:34:44,652 INFO    MainThread:3588 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-23 17:34:44,653 INFO    MainThread:3588 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-23 17:34:44,653 INFO    MainThread:3588 [wandb_init.py:init():533] updated telemetry
2022-02-23 17:34:44,766 INFO    MainThread:3588 [wandb_init.py:init():563] communicating current version
2022-02-23 17:34:46,419 INFO    MainThread:3588 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-23 17:34:46,420 INFO    MainThread:3588 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-23 17:34:46,671 INFO    MainThread:3588 [wandb_init.py:init():606] starting run threads in backend
2022-02-23 17:34:51,689 INFO    MainThread:3588 [wandb_run.py:_console_start():1810] atexit reg
2022-02-23 17:34:51,690 INFO    MainThread:3588 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-23 17:34:51,691 INFO    MainThread:3588 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-23 17:34:51,695 INFO    MainThread:3588 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-23 17:34:51,695 INFO    MainThread:3588 [wandb_init.py:init():633] run started, returning control to user process
2022-02-23 17:34:53,054 INFO    MainThread:3588 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-23 17:34:53,055 INFO    MainThread:3588 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-23 17:36:42,912 INFO    MainThread:3588 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-23 17:36:42,914 INFO    MainThread:3588 [wandb_run.py:_restore():1752] restore
2022-02-23 17:36:43,336 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-23 17:36:43,550 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-23 17:36:43,821 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25162
}

2022-02-23 17:36:43,926 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25162
}

2022-02-23 17:36:44,036 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 25162
}

2022-02-23 17:36:44,138 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25162
  total_bytes: 25162
}

2022-02-23 17:36:44,243 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25162
  total_bytes: 25162
}

2022-02-23 17:36:44,346 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25162
  total_bytes: 25162
}

2022-02-23 17:36:44,448 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25162
  total_bytes: 25162
}

2022-02-23 17:36:44,551 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25162
  total_bytes: 25162
}

2022-02-23 17:36:45,082 INFO    MainThread:3588 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 25162
  total_bytes: 25162
}

